roles/etcd/templates/etcd.env.j2

```diff
@@ -20,6 +20,9 @@ ETCD_SNAPSHOT_COUNT={{ etcd_snapshot_count }}
 {% if etcd_quota_backend_bytes is defined %}
 ETCD_QUOTA_BACKEND_BYTES={{ etcd_quota_backend_bytes }}
 {% endif %}
+{% if etcd_log_package_levels is defined %}
+ETCD_LOG_PACKAGE_LEVELS={{ etcd_log_package_levels }}
```

Comment:
could you define a default for `etcd_log_package_levels` please

Reply:
Yes, the etcd default is the omission of this flag, or an empty string "" (INFO for all packages)

I figured to follow the example right above this change with `ETCD_QUOTA_BACKEND_BYTES` which also does not have a default, which in a way makes sense since otherwise etcd was always started without the inclusion of that flag at all so it is likely the most expected behavior

Reply:
@woopstar Please let me know if my reasoning about omitting the default is incorrect, then I will be happy to add a `etcd_log_package_levels` in the role defaults